src/cli.rs

```diff
@@ -103,6 +103,10 @@ fn regenerate_local_binary(
         &parsed_config_file,
         has_local_rules,
         relative_path_from_local_binary_project_directory_to_project_directory,
+        parsed_config_file
+            .content
+            .tree_sitter_lint_dependency
```

Comment:
I guess this is a short-term solution for starting to run this against other actual project directories